src/scene/shader-lib/chunks/standard/frag/litShaderArgs.js

```diff
@@ -0,0 +1,85 @@
+export default /* glsl */`
+
+struct IridescenceArgs
+{
+    // The color of the fresnel effect, set by the backend
+    vec3 fresnel;
+
+    // Iridescence effect intensity, range [0..1]
+    float intensity;
+
+    // Thickness of the iridescent microfilm layer, value is in nanometers, range [0..1000]
+    float thickness;
+};
+
+struct ClearcoatArgs
+{
+    // Intensity of the clearcoat layer, range [0..1]
+    float specularity;
+
+    // Glossiness of clearcoat layer, range [0..1]
+    float gloss;
+
+    // The normal used for the clearcoat layer
+    vec3 worldNormal;
+};
+
+struct SheenArgs
+{
+    /// Glossiness of the sheen layer, range [0..1]
+    float gloss;
+
+    // The color of the f0 specularity factor for the sheen layer
+    vec3 specularity;
+};
+
+struct LitShaderArguments {
+    // Transparency
+    float opacity;
+
+    // Normal direction in world space
+    vec3 worldNormal;
+
+    // Surface albedo absorbance
+    vec3 albedo;
+
+    // Transmission factor (refraction), range [0..1]
+    float transmission;
+
+    // Uniform thickness of medium, used by transmission, range [0..inf]
+    float thickness;
+
+    // The f0 specularity factor
+    vec3 specularity;
+
+    // The microfacet glossiness factor, range [0..1]
+    float gloss;
+
+    // Surface metalness factor, range [0..1]
+    float metalness;
+
+    // Specularity intensity factor, range [0..1]
+    float specularityFactor;
+
+    // Ambient occlusion amount, range [0..1]
+    float ao;
```

Comment:
It seems this term is used in the chunks to occlude both diffuse and specular .. if I understand this correctly, is it correct to call it `ao` and not simply occlusion?

Reply:
It's debatable. We don't provide a specular occlusion term on the material, but we do however use the AO maps to occlude specularity as well.